scripts/globals/magic.lua

```diff
@@ -403,6 +403,10 @@ function getMagicHitRate(caster, target, skillType, element, percentBonus, bonus
 
     magicacc = magicacc + bonusAcc;
 
+    -- Add macc% from food
+    local maccFood = caster:getMod(MOD_FOOD_MACCP)
+    magicacc = magicacc + utils.clamp(maccFood, 0, MOD_FOOD_MACC_CAP);
```

Comment:
that line with the var was supposed to be `local maccFood = magicacc * (caster:getMod(MOD_FOOD_MACCP)/100);`